src/pseudopeople/noise_functions.py

```diff
@@ -614,6 +614,11 @@ def make_phonetic_errors(
         dataset.data[column_name].dtype,
     )
 
+    # make_phonetic_errors possibly introduces blanks when we noise
+    # some single-character values
+    became_blank = dataset.data.loc[to_noise_index, column_name] == ""
```

Comment:
What is this for? This makes the whole string blank?

Reply:
I was wondering whether empty string is treated the same as NaN throughout pseudopeople? Is it really appropriate to add this to `missingness`?